source/controllers/game.py

```diff
@@ -0,0 +1,78 @@
+from controller import controller
+from ..common import validator, encode_player_jwt, Response, Code, \
+    Status, Rule, Responses, TakiException, authenticated
+from ..games import create_game, join_game, leave_game, start_game
+
+
+@controller(Code.CREATE_GAME)
+@validator(Rule.CREATE_GAME)
+def create_game_controller(args, sock):
+    lobby_name = str(args['lobby_name'])
+    host = str(args['player_name'])
+    password = str(args['password'])
```

Comment:
Maybe strip every string to avoid small inconsistencies? Same goes for the other controllers

Reply:
Good idea